tla/tlc_debug.sh

```diff
@@ -6,7 +6,7 @@
 # The debug invariant(s) should be the only invariant(s), otherwise 
 # this script might falsely return without errors
 
-./tlc.sh "$@"
```

Comment:
This script could be removed and a `-expected_violation` flag could be added to `tlc.py`

Reply:
Will have a look

Reply:
https://github.com/tlaplus/tlaplus/issues/827#issuecomment-1675146379 shows ways how to assert the violation of "debug invariants" (that are also robust against checking true invariants).

Reply:
I think it'd be preferable to do what @lemmy suggests, than the current mechanism which looks quite specific to safety violations.